src/fidesops/service/connectors/post_processor_strategy/post_processor_strategy_filter.py

```diff
@@ -0,0 +1,78 @@
+import logging
+from typing import Any, Optional, Dict
+
+from fidesops.schemas.saas.strategy_configuration import (
+    FilterPostProcessorConfiguration,
+    StrategyConfiguration,
+    IdentityParamRef,
+)
+from fidesops.service.connectors.post_processor_strategy.post_processor_strategy import (
+    PostProcessorStrategy,
+)
+
+STRATEGY_NAME = "filter"
+
+logger = logging.getLogger(__name__)
+
+
+class FilterPostProcessorStrategy(PostProcessorStrategy):
+    """
+    Filters object or array given field name and value
+    Value can be reference a dynamic identity passed in through the request OR hard-coded value.
+    E.g.
+    data = [
+        {
+            id: 1397429347
+            email_contact: somebody@email.com
+            name: Somebody Awesome
+        },
+        {
+            id: 238475234
+            email_contact: somebody-else@email.com
+            name: Somebody Cool
+        }
+    ]
+    field: email_contact
+    value: {identity: email}, where email == somebody@email.com
+    result = {
+        id: 1397429347
+        email_contact: somebody@email.com
+        name: Somebody Awesome
+    }
+    """
+
+    def __init__(self, configuration: FilterPostProcessorConfiguration):
+        self.field = configuration.field
+        self.value = configuration.value
```

Comment:
So you're saying when `configuration.value` is a string `"123"`, and we're filtering data where the field is int `123`? 

This could definitely be an issue. We can either coerce to str or include an additional data type param. For now, I'm thinking we can coerce to str and add this to docs?

Reply:
ok I'll aim for that, thanks!

Reply:
Making a note that we'll follow up with adding type cast functionality in a separate ticket

Reply:
New ticket here: https://github.com/ethyca/fidesops/issues/265

Reply:
thanks for reticketing! you were bringing up examples I hadn't thought of myself.